### ISSUE TITLE
No Scrolling

### ISSUE THREAD

**Issue by ashokbatesar** [RangerMauve/react-native-square-grid#7]
How Can I add Scrollbar in this grid. like i want to make a gallery album.

**Comment by RangerMauve** [comment]
Yes, if you set `rows` to `false` or `0` it'll try to render as many as possible and scroll.
